src/__stories__/list-story.tsx

```diff
@@ -96,17 +99,31 @@ const Template: StoryComponent<Args & {boxed?: boolean}> = ({
                 break;
             case 'custom element':
                 controlProps = {
-                    right: (
+                    right: () => (
                         <div style={{display: 'flex', alignItems: 'center', height: '100%'}}>
                             <div style={{width: 32, height: 32, borderRadius: '50%', background: 'pink'}} />
                         </div>
```

Comment:
Please, use the callback parameter in the example

Reply:
I kept this example as a React.Node instead the callback function, so we have both examples, with and without the new right callback with centerY.